src/Entity/Conference.php

```diff
@@ -116,6 +111,16 @@ class Conference
      */
     private $participations;
 
+    /**
+     * @ORM\Column(type="string", length=255)
+     */
+    private $country;
```

Comment:
```suggestion
    private ?string $country = null;
```

same after. We are using PHP 7.4, let's use it :)